.env.example

```diff
@@ -0,0 +1,4 @@
+DECAF_JS_SKIP_POSTINSTALL=1
+TESTING_API_KEY=
```

Comment:
This is where we read our environment variables. Repository secrets with the same name are created for Github Actions.